### DISCUSSION TITLE
Can anyone explain how to get these??? Plssss

### DISCUSSION THREAD

**Discussion by ErikaXGH** [valinet/ExplorerPatcher#3934] [General]
![1815B449-A810-4547-87BE-9DAF3EC02D5C.png](https://github.com/user-attachments/assets/bbf08f8a-dd6c-491d-9b16-218f9d922d61)

**Comment by pyrates999** [comment]
the taskbar changes?  Install translucentTB or NiceTaskBar.  Available from Microsoft App Store.

**Comment by milestprower92** [comment]
Install [Windhawk](https://www.windhawk.net/download) and use windows 11 taskbar styler
